flowey/flowey_lib_hvlite/src/artifact_openvmm_hcl_sizecheck.rs

```diff
@@ -48,3 +48,52 @@ pub mod publish {
         }
     }
 }
+
+/// Resolve the contents of an existing artifact.
+pub mod resolve {
+    use crate::build_openvmm_hcl::OpenvmmHclOutput;
```

Comment:
might be better to use a distinct type to represent the contents of this artifact, given that we don't expect to ever have the resulting OpenvmmHclOutput to have Some dbg field